tools/local-network/local_network.py

```diff
@@ -29,16 +28,19 @@
 IAS_SPID = os.getenv('IAS_SPID', default='0'*32) # 16 bytes
 PROJECT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
 MOB_RELEASE = os.getenv('MOB_RELEASE', '1')
-CARGO_FLAGS = '--release'
-TARGET_DIR = 'target/release'
+if MOB_RELEASE == '1':
+    CARGO_FLAGS = '--release'
+    BUILD_TYPE = 'release'
+else:
+    CARGO_FLAGS = ''
+    BUILD_TYPE = 'debug'
+
+BASE_TARGET_DIR = os.getenv('CARGO_TARGET_DIR', 'target')
+TARGET_DIR = os.path.join(os.path.relpath(BASE_TARGET_DIR, PROJECT_DIR), BUILD_TYPE)
```

Comment:
I think going back and forth from relative to absolute paths is confusing.
How about something like this?
```
PROJECT_TARGET_DIR = os.path.join(PROJECT_DIR, 'target') # The default target directory
BASE_TARGET_DIR = os.path.abspath(os.getenv('CARGO_TARGET_DIR', PROJECT_TARGET_DIR)) # Allow overriding with the `CARGO_TARGET_DIR` env var
TARGET_DIR = os.path.join(BASE_TARGET_DIR, BUILD_TYPE) # Append `debug`/`release`
WORK_DIR = os.path.join(TARGET_DIR, 'mc-local-network')
```

Reply:
Should be similar